docs-2.0/synchronization-and-migration/replication-between-clusters.md

```diff
@@ -386,9 +386,11 @@ The schema data is synchronized through the Meta listener, while the vertex/edge
 
 ### How to deal with synchronization failures?
 
-- If problems happen on the primary cluster, the synchronization will be paused. Fixing the problems and then restarting the primary cluster can continue the synchronization.
+Fix the problems in the cluster, and the synchronization will be automatically restored.
 
-- If problems happen on the secondary cluster, listeners or drainers, when the problem is fixed, the services that had the problems will receive the WALs accumulated from its upstream and the synchronization will continue. If the faulty machine is replaced with a new one, all the data of the synchronization services on the faulty machine must be copied to the new machine. Otherwise, the synchronization of the complete data set starts.
+- If problems happened on the primary cluster, the synchronization would continue when the problems were fixed and the primary cluster restarted.
```

Comment:
```suggestion
- If problems have happened on the primary cluster, the synchronization continues when the problems are fixed and the primary cluster restarts.
```

Reply:
Great! Thanks for reminding me about how to write this kind of content!